docs/arcaflow/workflows/flow-control.md

```diff
@@ -107,3 +107,182 @@ outputs:
     my_plugin: !expr $.steps.my_plugin.outputs.success
     my_other_plugin: !expr $.steps.my_other_plugin.outputs.success
 ```
+
+### Dry Out with bindConstants()
+
+The builtin function [bindConstants()](expressions.md#functions) allows you to factor out any constant input variables to a foreach subworkflow. The input variable `name`'s value is repeated across each iteration in this input. This section will show you how to simplify it.
+
+```yaml title="input-repeat-name.yaml"
+iterations:
+  - loop_id: 1
+    name: Matt
+  - loop_id: 2
+    name: Matt
+  - loop_id: 3
+    name: Matt
+  - loop_id: 4
+    name: Matt
+```
+
+```yaml title="workflow-repeat-name.yaml"
+version: v0.2.0
+input:
+  root: RootObject
+  objects:
+    RootObject:
+      id: RootObject
+      properties:
+        iterations:
+          type:
+            type_id: list
+            items:
+              type_id: object
+              id: SubRootObject
+              properties:
+                loop_id:
+                  type:
+                    type_id: integer
+                name:
+                  type:
+                    type_id: string
+
+steps:
+  foreach_loop:
+    kind: foreach
+    items: !expr $.input.iterations
+    workflow: subworkflow-repeat-name.yaml
+    parallelism: 1
+
+outputs:
+  success:
+    fab_four: !expr $.steps.foreach_loop.outputs.success.data
+```
+
+```yaml title="subworkflow-repeat-name.yaml"
+version: v0.2.0
+input:
+  root: SubRootObject
+  objects:
+    SubRootObject:
+      id: SubRootObject
+      properties: 
+        name:
+          type:
+            type_id: string
+steps:
+  example:
+    plugin:
+      src: quay.io/arcalot/arcaflow-plugin-template-python:all_d5dd9d6
+      deployment_type: image
+    input:
+      name: !expr $.input.name
+
+outputs:
+  success:
+    loop_id: !expr $.input.item.loop_id
+    beatle: !expr $.steps.example.outputs.success
+```
+
+#### Dried Out Workflow
```

Comment:
I don't know that "Dried Out" is really a good header. It refers loosely to the DRY principle, but that's *not* the same as the English word "dry" with a past tense "dried". Webb refers in a later comment to "DRYed", but I'm not sure I like that much either. How about twinsie H4 headers "the old way" and "the new way", or "without `bindConstants`" and "with `bindConstants`"?

Reply:
"Old" vs. "new" would probably be better; however, at some point, the "new" way will have been around for awhile and won't really be "new" anymore.

Perhaps, "simplified" vs. "complex"; or, "explicit" or "manual" vs. "automatic"; or...

Reply:
i went with 'Reduce Repetition'

Reply:
oh i missed that spot